subsys/bluetooth/host/conn.c

```diff
@@ -3067,6 +3078,18 @@ void bt_hci_le_df_connection_iq_report(struct net_buf *buf)
 
 	bt_conn_unref(conn);
 }
+
+void bt_hci_le_df_connection_iq_report(struct net_buf *buf)
+{
+	bt_hci_le_df_connection_iq_report_common(BT_HCI_EVT_LE_CONNECTION_IQ_REPORT, buf);
+}
+
+#if defined(CONFIG_BT_DF_VS_CONN_IQ_REPORT_16_BITS_IQ_SAMPLES)
+void bt_hci_le_vs_df_connection_iq_report(struct net_buf *buf)
+{
+	bt_hci_le_df_connection_iq_report_common(BT_HCI_EVT_VS_LE_CONNECTION_IQ_REPORT, buf);
```

Comment:
I guess you're missing something here: you send the same event in both cases.

Reply:
No ;-) `BT_HCI_EVT_LE_CONNECTION_IQ_REPORT` != `BT_HCI_EVT_VS_LE_CONNECTION_IQ_REPORT`

Reply:
yep misread